ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -129,7 +129,7 @@ elif [ "$PX4_SIMULATOR" = "gz" ] || [ "$(param show -q SIM_GZ_EN)" = "1" ]; then
 			exit 1
 		fi
 
-	elif [ -n "${PX4_GZ_MODEL_NAME}" ] && [ -z "${PX4_GZ_MODEL}" ]; then
+	elif [ -n "${PX4_GZ_MODEL_NAME}" ] && [ -z "${PX4_SIM_MODEL#*gz_}" ]; then
```

Comment:
In this case you want to attach to an already existing model. The coditions are

1. `PX4_GZ_MODEL_NAME` is not empty.
2. `PX4_SIM_MODEL#*gz_` is empty.

However, `PX4_SIM_MODEL#*gz_`  is never empty as all models define it if it is not already defined: https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/init.d-posix/airframes/4001_gz_x500#L12

Therefore this condition is never met and the bridge fails to start.

```suggestion
	elif [ -n "${PX4_GZ_MODEL_NAME}" ]; then
```